proxies/live/apiproxy/policies/AssignMessage.Remove.x-ers-user-id-header.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.Remove.x-ers-user-id-header">
+  <Remove>
```

Comment:
Could this removal be completed as part of the current AssignMessage policy rather than being added separately? Just thinking it might ensure that one policy can't be included without the other.